src/ui/components/entityregions/EntityRegions.js

```diff
@@ -49,7 +49,7 @@ const EntityRegions = (props) => {
     const gridDimensions = {
         xs: 12,
         sm: 12,
-        md: showEqpTree || regionMaximized ? 12 : 12 / Object.keys(columns).length,
+        md: 12,
```

Comment:
Why was this changed?

Reply:
This was changed to be able to fit the table cleanly at all resolutions.